tests/e2e/Services/Functions/FunctionsBase.php

```diff
@@ -18,7 +18,7 @@ protected function packageCode($folder)
     protected function awaitDeploymentIsBuilt($functionId, $deploymentId, $checkForSuccess = true): void
     {
         $startTime = time();
-        $maxWaitTime = 15;
+        $maxWaitTime = 40;
```

Comment:
why did we increase this? Which deployments take longer than 15 seconds?

Reply:
`Ruby` and `Python` runtime tests for some versions take longer than 15 seconds to build.